frontend/src/metabase/admin/upsells/components/Upsells.module.css

```diff
@@ -0,0 +1,23 @@
+:root {
+  --mb-color-upsell-primary: #005999;
+  --mb-color-upsell-secondary: #bff4ff;
+  --mb-color-upsell-gem: #00d4ff;
+}
+
+.UpsellBannerComponent {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 1rem;
+  padding: 1rem;
+  overflow: hidden;
+  border: 1px solid var(--mb-color-border);
+  border-radius: 0.5rem;
+}
+
+.SecondaryCTALink {
+  display: inline;
+  color: var(--mb-color-upsell-primary);
+  font-weight: 700;
+  margin-left: 0.25rem;
+}
```

Comment:
we probably want a hover state on the link component

Reply:
I'll check with Roman if he has any preferences and will add it as part of the next PR (we'll use this in the UpsellBigComponent).